### PR TITLE
Adds two numbers using only [, ], (, ), +, and !

### PR DESCRIPTION
**Pull request by diiq**
This addition function is highly efficient for developers, because it only requires six characters to type! This saves much wear and tear on keyboards and on fingers.

### CODE REVIEW

File: adds_functions.js [hunk 1]
```diff
@@ -16,5 +16,11 @@ module.exports = {
   frankSubtractTwoNumbers: function(numberOne, numberTwo){
     return numberOne - numberTwo;
   },
-  
+
+  samsAddTwoNumbers: new Function('e', 't',
```
Comment (kendelljoseph) [inline adds_functions.js:20]: Hahahahah! -- This has to be the most ridiculous thing I have ever seen!

File: features/check_adds_functions.feature [hunk 2]
```diff
@@ -29,4 +29,11 @@ Feature: These are functions that add two numbers
       And we want to use 0 as the first number
       And we want to use 0 as the second number
     When we add our numbers using the frankSubtractTwoNumbers function
-    Then the result should equal 0
\ No newline at end of file
+    Then the result should equal 0
+
+  Scenario: A person wants to add two numbers together using Sam's Function
```
Comment (kendelljoseph) [inline features/check_adds_functions.feature:34]: 👍

**Review by kendelljoseph — APPROVED** [review]
Let there be code!

**Comment by kendelljoseph** [comment]
Down to the name of the branch, this is just the worst!!!

File: adds_functions.js (hunk 1 above)
Comment (diiq) [reply adds_functions.js:20]: Haha, it's so rare to get an opportunity to pull out the weird stuff. It was a good reminder to just have fun and play!

**Pull request merged**